hooks/useERC721Transfer.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from "react"
+import { transferFromERC721 } from "../lib/cre8or"
+import { useEthersSigner } from "./useEthersSigner"
+
+const useERC721Transfer = ({ afterTransfer }) => {
+  const signer = useEthersSigner()
+
+  const transferERC721 = useCallback(
```

Comment:
does `transferERC721` need a `useCallback`?

what does `useCallback` do here?

can it be removed?